nautobot/users/views.py

```diff
@@ -213,6 +226,118 @@ def post(self, request):
         )
 
 
+#
+# Saved Views
+#
+
+
+class SavedViewUIViewSet(
+    ObjectDetailViewMixin,
+    ObjectBulkDestroyViewMixin,
+    ObjectChangeLogViewMixin,
+    ObjectDestroyViewMixin,
+    ObjectEditViewMixin,
+    ObjectListViewMixin,
+):
+    queryset = SavedView.objects.all()
+    filterset_class = SavedViewFilterSet
+    table_class = SavedViewTable
+    action_buttons = ("export",)
+
+    def get_table_class_string_from_view_name(self, view_name):
+        model = get_model_for_view_name(view_name)
+        if model:
+            return get_table_for_model(model)
```

Comment:
Not sure if this was an intentional change in the latest commits, but this function is now returning a table class object instead of a string, which at least doesn't match the method name.

Reply:
Oops thank you for catching that!